### ISSUE TITLE
Satellite library header locations?

### ISSUE THREAD

**Issue by slouken** [libsdl-org/SDL#8062]
Right now we place headers for satellite libraries in the SDL3 subdirectory, but when building an Apple Framework, they need to be in a directory named for the framework.

For example, we recommend this:
`#include <SDL3/SDL_image.h>`
but on Apple platforms it needs to be:
`#include <SDL3_image/SDL_image.h>`

I ran into this when trying to build the test applications under Xcode. I realized that we don't include the test headers in SDL.framework (good) but it means that in order to make the test library easily available to applications, I'd have to create a new framework and install the headers under that, and would have to move the headers from SDL3 to SDL3_test and change the includes to `#include <SDL3_test/SDL_test.h>`

@madebr, can you help move the headers appropriately?

**Comment by slouken** [comment]
Tagging @sezero for visibility

**Comment by madebr** [comment]
Are you going to ship a SDL3_test binary framework?
If so, it will also need cmake support (pkg-support folder).

**Comment by slouken** [comment]
No, sorry, I removed that portion of the request, but it looks like you already started on it. I'm working on the tests now, and won't be building a separate SDL3_test framework, or need the headers to be moved. This request is just regarding the satellite libraries.

**Comment by sezero** [comment]
> Tagging @sezero for visibility

I'm not sure what is asked of me?

BTW, if this is to be done here, other satellite libs will need the same.

**Comment by slouken** [comment]
> I'm not sure what is asked of me?

Nothing specific, I just wanted to make sure you were aware of this requirement.

**Comment by smcv** [comment]
This has been done for image, mixer, ttf and rtf.

Are there any other satellite libraries that are still relevant?

(If I understand correctly, SDL_net is dead and will not be getting a SDL3 port? It's IPv4-only.)

**Comment by slouken** [comment]
We're not currently planning an SDL3 port of SDL_net, although Maelstrom still uses it, so that may change. :)
